src/main.ts

```diff
@@ -21,6 +21,8 @@ export async function main(): Promise<Outputs> {
     const deploymentMode = getInput('deploymentMode')
     const deploymentName = getInput('deploymentName')
     const parameters = getInput('parameters')
+    const managementGroupId = getInput('managementGroupId')
+    const validationOnly = getInput('validationOnly') == 'true';
```

Comment:
IMO default value should be false as primary usage of action would be to deploy.

Reply:
Ain't it false by default, as you have to explicitly specify the value `true`

Reply:
yes, you are right.